### PR TITLE
Doc: popper check removed from getting started guide.

### PR DESCRIPTION
**Pull request by UjjwalAyyangar**
Addresses #267 

* Removed instructions for popper check since it has not been added yet to the python version
* Added instructions on how to run a popper pipeline.

### CODE REVIEW

File: docs/protocol/getting_started.md [hunk 1]
```diff
@@ -62,24 +62,10 @@ git add pipelines/myexp
...
-this, one can create a pipeline using the `--env` flag of the `popper 
-init` command. For example, `popper init <pipeline> --env=alpine-3.4` 
-runs a command inside an `alpine-3.4` container.
-
-Once a pipeline is checked, one can show the logs:
+### Run a pipeline
 
 ```bash
-ls -l pipelines/myexp/popper_logs
+popper run myexp
```
Comment (ivotron) [inline docs/protocol/getting_started.md]: This was fine, the only issues is that `popper check` is now named `popper run`.

Comment (UjjwalAyyangar) [reply docs/protocol/getting_started.md]: @ivotron I thought we would be implementing some different features in the popper check command. I think I misunderstood, I will make the required changes.

Comment (UjjwalAyyangar) [reply docs/protocol/getting_started.md]: Completed.

**Comment by ivotron** [comment]
thanks!

**Pull request merged**